### PR TITLE
Add luamocks for better IDE support

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: luamocks/LuaSyncedRead.lua [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU General Public License for more details.
+-- You should have received a copy of the GNU General Public License
+--along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--==================================================================================================
+--Variables for the MockUp
+
+Spring ={}
+numberMock =42
+stringMock =TestString
```
Comment (sprunk) [inline luamocks/LuaSyncedRead.lua:18]:
```suggestion
stringMock ="TestString"
```

Comment (badosu) [reply luamocks/LuaSyncedRead.lua]: Done, thanks!

**Comment by Ruwetuin** [comment]
fow what IDE exactly would this be useful?

**Comment by badosu** [comment]
Any that has support for LSP: https://microsoft.github.io/language-server-protocol/implementors/tools/

**Pull request merged**